<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#MainWindow{ background-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.5, fx:0.5, fy:0.5, stop:0 rgba(255, 0, 0, 255), stop:0.479904 rgba(255, 0, 0, 255), stop:0.522685 rgba(255, 255, 255, 255), stop:1 rgba(255, 255, 255, 255)) }



#Primary{
color: #fff;
background-color: #0055ff;
}
#Primary:hover{
 background-color: #0055ff;
}
#Primary:pressed{border: 4px solid #98c1fe}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>360</y>
      <width>391</width>
      <height>171</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>373</width>
       <height>167</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QRadioButton" name="initNode">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Инициализация</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="addNode">
          <property name="text">
           <string>Добавить узел</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="delNode">
          <property name="text">
           <string>Удалить узел</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QRadioButton" name="arr">
        <property name="text">
         <string>Последовательность узлов</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLineEdit" name="userData"/>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton">
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="acceptDrops">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">#Primary{
color: #fff;
background-color: #0055ff;
}
#Primary hover{
 background-color: #0055ff;
}
#Primary:pressed{border: 4px solid #98c1fe}
</string>
          </property>
          <property name="text">
           <string>Применить</string>
          </property>
          <property name="shortcut">
           <string/>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="showBtn">
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="text">
           <string>Показать</string>
          </property>
          <property name="autoDefault">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGraphicsView" name="grView">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>21</y>
      <width>741</width>
      <height>341</height>
     </rect>
    </property>
    <property name="cursor" stdset="0">
     <cursorShape>CrossCursor</cursorShape>
    </property>
    <property name="backgroundBrush">
     <brush brushstyle="NoBrush">
      <color alpha="255">
       <red>86</red>
       <green>126</green>
       <blue>147</blue>
      </color>
     </brush>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
